Simplified.xcodeproj/project.pbxproj

```diff
@@ -4298,250 +4274,6 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
-		7347F037245A4DE200558D7F /* Sources */ = {
```

Comment:
It looks like it happened during rebasing my current commit on the latest develop branch

Reply:
I checked that key on develop and it's not there anymore, so I think this just means your feature branch is behind and your `task/OE-14` branch includes more changes from `develop`. This is a bit unusual but it's ok. 

The more usual git-flow process does this instead:
- routinely merge `develop` on the feature branch
- start or rebase your new commit branches (like `task/OE-14`) on top of the _feature_ branch, not `develop`.

This also reduces the amount of changes in PR -- for example this would not have showed up here.